README.md

```diff
@@ -30,8 +31,10 @@
 
 ## Options for working with this material
 
-- [Browser-based setup with GitPod](https://testingaccessib-workshop-abielzjxum9.ws-us39a.gitpod.io)
-  - Note: GitPod may have accessibility issues
+- [Browser-based setup with GitPod](https://gitpod.io/from-referrer/)
```

Comment:
using https://gitpod.io/from-referrer/ makes it simple here to capture the current repo and spin up a dev environment for the user (versus the link above that points to an instance that the user likely doesn't have access to)